deploy-satellite-update-playbook.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I agree with removing "deploy" from the names of the playbook.  But I like to keep "playbook" suffix so it is obvious and easy for the user to find the right playbook.